superset/assets/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -101,6 +102,34 @@ class SliceHeaderControls extends React.PureComponent {
     });
   }
 
+  downloadChartImage(ev){
```

Comment:
This looks copy pasted from the other file, can you pull it out into another file and import it in both components?